quiz/generic/quiz_defaults.inc

```diff
@@ -58,32 +57,27 @@ $messages["P_commascanno"] = array(
 );
 $messages["P_accentbracket"] = array(
     "message_title" => _("Accent"),
-    "message_body" => _("Where possible, proofread accented letters using the actual characters, like <kbd>&aacute;</kbd>. You can insert accented letters using the character picker below."),
+    "message_body" => _("Where possible, proofread accented letters using the actual characters, like <kbd>á</kbd>. You can insert accented letters using the character picker below."),
     "guideline" => "a_chars"
 );
 $messages["P_oe"] = array(
     "message_title" => _("Ligature"),
-    "message_body" => _("Proofread the 'oe' ligature (&oelig;) using the symbol directly, not as <kbd>oe</kbd> or <kbd>[oe]</kbd>. You can insert the symbol using the character picker below."),
+    "message_body" => _("Proofread the 'oe' ligature (<kbd>œ</kbd>) using the symbol directly, not as <kbd>oe</kbd> or <kbd>[oe]</kbd>. You can insert the symbol using the character picker below."),
     "guideline" => "a_chars"
 );
 $messages["P_macron"] = array(
     "message_title" => _("Macron"),
-    "message_body" => _("Following the proofreading guidelines, please use the appropriate Unicode character for a letter with a macron (straight line) over it.  For instance, proofread an '<kbd>a</kbd>' with a macron over it as <kbd>&#257;</kbd>.  If the macron appears over an '<kbd>e</kbd>' it would be <kbd>&#275</kbd>, and so forth.") . " " . _("If the project's character suites do not include the character, use the diacritical markup as described in the Proofreading Guidelines."),
+    "message_body" => _("Following the proofreading guidelines, please use the appropriate Unicode character for a letter with a macron (straight line) over it.  For instance, proofread an '<kbd>a</kbd>' with a macron over it as <kbd>ā</kbd>.  If the macron appears over an '<kbd>e</kbd>' it would be <kbd>ē</kbd>, and so forth. If the project's character suites do not include the character, use the diacritical markup as described in the Proofreading Guidelines."),
     "guideline" => "d_chars"
 );
 $messages["P_aebracket"] = array(
     "message_title" => _("Ligature"),
-    "message_body" => _("Proofread the 'ae' ligature (<kbd>&aelig;</kbd>) using the symbol directly; brackets are not necessary for this character. You can insert the symbol using the character picker below."),
+    "message_body" => _("Proofread the 'ae' ligature (<kbd>æ</kbd>) using the symbol directly. You can insert the symbol using the character picker below."),
```

Comment:
I do not want to refer to a specific menu as that can get out of sync.

I think we've overloaded the term a bit. The "character picker" is the entire character selection widget. In the code we call the selections available in a menu as "character picker" as well. If we don't want to call the whole widget "character picker" then we need to pick a new name to call it.

Reply:
That's fair.

I tend to think of each individual picker as a character picker, and the picker area as containing pickersets, but that is informal. I mostly wanted to be as precise as we can to avoid confusion, but that may be unavoidable.